docs/source/reference/links.rst

```diff
@@ -191,6 +191,13 @@ where ``fc7`` denotes a layer before the last fully-connected layer.
 Unlike the usual links, these classes automatically load all the
 parameters from the pre-trained models during initialization.
 
+GoogLeNet
+~~~~~~~~~
+.. autoclass:: GoogLeNet
+   :members:
+
+.. autofunction:: chainer.links.model.vision.googlenet.prepare
+
```

Comment:
I'm curious why you decided to place `GoogLeNet` at the top instead of `VGG16Layers`.
My feeling is that VGG model is the most commonly used, and GoogLeNet is the next.

If you think that these models should be arranged alphabetically, it seems better to rearrange the other models.